core/arch/arm/plat-imx/imx-common.c

```diff
@@ -19,15 +19,18 @@ static int imx_soc_revision = (-1);
 static void imx_digproc(void)
 {
 	uint32_t digprog;
-	vaddr_t anatop_addr;
+	vaddr_t __maybe_unused anatop_addr;
```

Comment:
by the way, could you default init these 2 to something (i.e.`0`)

Reply:
Addressed in a5496ac